CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## 2017-06-02
```

Comment:
Sorry I was not clear enough, what I wanted was:
```
### Fixed
- check-postgresq-replication.rb: Adds missing option for custom port.
```

In this case I will just fix it for you but in general we leave all that stuff to the maintainer since there is no guarantee on the order of reviews/merges (not exactly FIFO) or when it gets reviewed and merged.